### DISCUSSION TITLE
non-redundant versus irredundant

### DISCUSSION THREAD

**Discussion by arminbiere** [ipasir2/ipasir2#32] [General]
As the literature (yes I am biased) starting with our IJCAR'12 inprocessing paper used the term 'irredundant' as negation of 'redundant', I would suggest to stick to that and replace 'non-redundant' with 'irredundant' throughout documents and code of IPASIR for consistency.

**Comment by Udopia** [comment]
Alright, entirely sticking to irredundant as of commit f2a9a11aa8ca9f7c922a94d7471fa9e79cf5e82b
